components/script/dom/workerlocation.rs

```diff
@@ -34,6 +34,11 @@ impl WorkerLocation {
             WorkerLocationBinding::Wrap,
         )
     }
+
+    // https://html.spec.whatwg.org/multipage/workers.html#dom-workerlocation-origin
+    pub fn origin(&self) -> &MutableOrigin {
```

Comment:
This module didn't import `MutableOrigin` but you use it here.
So, I believe you'll want to import it.

Reply:
oops, thanks :)